### DISCUSSION TITLE
Charts Not Displaying Correctly After Upgrading to Livewire v3.5.6

### DISCUSSION THREAD

**Discussion by sellouter-dev** [leandrocfe/filament-apex-charts#128] [Q&A]
After upgrading Livewire to version v3.5.6, the charts in my Laravel application are no longer rendering correctly. The charts either fail to appear, display distorted data, or do not update dynamically as expected.

![Capture-2024-09-20-230837](https://github.com/user-attachments/assets/ff204230-d4ee-4551-8b64-ce2802d36575)

![Capture-2024-09-20-230938](https://github.com/user-attachments/assets/5e97b958-484c-453d-ac47-2aef79efd75a)

**Comment by garniergeorges** [comment]
Hi,

I encountered this problem yesterday as well, you can disabling lazy loading.

`protected static bool $isLazy = false;`

I put it on my widgets and it worked.

**Comment by melvinmihaylovnulabg** [comment]
Hey guys, 
I have the same issue. 

isLazy to false did not work, how can i fix it?
